src/event.functions/eventSignup.js

```diff
@@ -128,26 +114,13 @@ exports.main = ({ body, accountId }, sendResponse) => {
 
   const updateContact = async () => {
     const formApiWithGuid = `${FORMS_API}/${accountId}/${formId}`;
-
-    const { statusCode, body } = await request({
-      method: 'POST',
-      json: true,
-      simple: true,
-      uri: formApiWithGuid,
+    
+    const { status, data } = await axios.post(formApiWithGuid, {
       body: {
         fields: [
-          {
-            name: 'firstname',
-            value: firstName,
-          },
-          {
-            name: 'lastname',
-            value: lastName,
-          },
-          {
-            name: 'email',
-            value: email,
-          },
+          firstName,
```

Comment:
I think that the forms API might be expecting data in the `{name, value}` format

Reply:
My bad, should be fixed in latest commit 😄